### ISSUE TITLE
Spectator with board cause error.

### ISSUE THREAD

**Issue by ryush00** [vram4/Ogar#71]
Clients:

1: player
2: Spector

```
>board test broadcast
[Console] Successfully changed leaderboard values
>/root/Ogar/src/PlayerTracker.js:251
        if (specPlayer.cells.length == 0) {
                            ^
TypeError: Cannot read property 'length' of undefined
    at PlayerTracker.getSpectateNodes (/root/Ogar/src/PlayerTracker.js:251:29)
    at PlayerTracker.calcViewBox (/root/Ogar/src/PlayerTracker.js:208:21)
    at PlayerTracker.update (/root/Ogar/src/PlayerTracker.js:111:31)
    at GameServer.updateClients (/root/Ogar/src/GameServer.js:312:39)
    at GameServer.mainLoop (/root/Ogar/src/GameServer.js:281:14)
    at wrapper [as _onTimeout] (timers.js:265:14)
    at Timer.listOnTimeout (timers.js:110:15)
```

I'm looking for problem.

**Comment by Ephemerality** [comment]
Problem is that spectator mode for FFA relies on the leaderboard for pulling players to spectate. Possible fix would be to add a line in the "board" command that sets gameServer.gameMode.specByLeaderboard to false.

**Comment by ryush00** [comment]
```
>board test
[Console] Successfully changed leaderboard values
>boardreset
[Console] Successfully reset leaderboard
>/root/Ogar/src/PlayerTracker.js:251
        if (specPlayer.cells.length == 0) {
                            ^
TypeError: Cannot read property 'length' of undefined
    at PlayerTracker.getSpectateNodes (/root/Ogar/src/PlayerTracker.js:251:29)
    at PlayerTracker.calcViewBox (/root/Ogar/src/PlayerTracker.js:208:21)
    at PlayerTracker.update (/root/Ogar/src/PlayerTracker.js:111:31)
    at GameServer.updateClients (/root/Ogar/src/GameServer.js:314:39)
    at GameServer.mainLoop (/root/Ogar/src/GameServer.js:283:14)
    at wrapper [as _onTimeout] (timers.js:265:14)
    at Timer.listOnTimeout (timers.js:110:15)
```

@vram4 @Ephemerality It seems not fixed fully. When I type ```boardreset```, server is crashed.

**Comment by vram4** [comment]
I pushed a fix for this a809cd93b22c8ef4c90ea16708ab525e90cbf1ae.
